pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "Cython ~=0.29",
+    "Cython >=0.29.31",
```

Comment:
I'm a bit wary of declaring Cython 3 compatibility before the final version is out (I think the `~=0.29` specifier is equivalent to `>=0.29, <1`). Though I see they're onto betas now, so hopefully it's not far off.

Reply:
maybe `>=0.29.31, <1` ?  We need to put a floor on the patch version so that the keyword exists.

Reply:
Yup, if we do that, I think it's fine.

Reply:
done.